specification/communication/resource-manager/Microsoft.Communication/preview/2023-03-01-preview/SenderUsernames.json

```diff
@@ -0,0 +1,333 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Communication Services Management API",
+    "version": "2023-03-01-preview",
+    "description": "REST API for Email Services/Domains/SenderUsernames"
+  },
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Communication/emailServices/{emailServiceName}/domains/{domainName}/senderUsernames": {
+      "get": {
+        "description": "List all valid sender usernames for a domains resource.",
+        "operationId": "SenderUsernames_ListByDomains",
+        "summary": "ListBy_Domains",
+        "tags": [
+          "SenderUsernames"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "EmailServices.json#/parameters/EmailServiceName"
+          },
+          {
+            "$ref": "Domains.json#/parameters/DomainName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success. The response describe the Domains SenderUsername collection.",
+            "schema": {
+              "$ref": "#/definitions/SenderUsernameResourceCollection"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
```

Comment:
>v3 [](http://example.com/codeflow?start=71&length=2)

Latest version of common types types.json is v5

Reply:
Sudipto (@usvoyager ), this seems ok but let me know if you think this may be a concern